### PR TITLE
Poc trace reporter

### PR DESCRIPTION
**Pull request by skolomiiets**

### CODE REVIEW

File: reactor-aeron-benchmarks/src/main/java/reactor/aeron/AeronPingClient.java [hunk 1]
```diff
@@ -105,19 +110,41 @@ private static void roundTripMessages(AeronConnection connection, long count) {
...
   private static void report(Object ignored) {
-    System.out.println("---- PING/PONG HISTO ----");
-    HISTOGRAM.getIntervalHistogram().outputPercentileDistribution(System.out, 5, 1000.0, false);
-    System.out.println("---- PING/PONG HISTO ----");
+    reporter.dumpTo("./target/traces/");
+
+    // System.out.println("---- PING/PONG HISTO ----");
+    // HISTOGRAM.getIntervalHistogram().outputPercentileDistribution(System.out, 5, 1000.0, false);
+    // System.out.println("---- PING/PONG HISTO ----");
+  }
```
Comment (artem-v) [inline reactor-aeron-benchmarks/src/main/java/reactor/aeron/AeronPingClient.java]: Uncomment lines 134-136

File: reactor-aeron-benchmarks/src/main/java/reactor/aeron/AeronPingClient.java [hunk 2]
```diff
@@ -105,19 +110,41 @@ private static void roundTripMessages(AeronConnection connection, long count) {
...
+  private static void collect(Object ignored) {
+    Histogram h = HISTOGRAM.getIntervalHistogram();
+
+    reporter.addY("reactor-aeron-latency-mean", h.getMean() / 1000.0);
+    reporter.addY("reactor-aeron-latency-99p", h.getPercentileAtOrBelowValue(99) / 1000.0);
+
+    // System.out.println("---- PING/PONG HISTO ----");
+    // HISTOGRAM.getIntervalHistogram().outputPercentileDistribution(System.out, 5, 1000.0, false);
+    // System.out.println("---- PING/PONG HISTO ----");
   }
```
Comment (artem-v) [inline reactor-aeron-benchmarks/src/main/java/reactor/aeron/AeronPingClient.java]: Uncomment lines 145-147

File: reactor-aeron-benchmarks/src/main/java/reactor/aeron/ServerThroughput.java [hunk 3]
```diff
@@ -4,6 +4,7 @@
```
Comment (artem-v) [inline reactor-aeron-benchmarks/src/main/java/reactor/aeron/ServerThroughput.java]: Revert changes in this file

**Comment by ronenhamias** [comment]
@artem-v its still in POC phase and subject to many changes

**Pull request closed without merging**